test/lib/sealField.test.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const assert = require('chai').assert;
+const mockery = require('mockery');
+const sinon = require('sinon');
+const symbolField = Symbol('symbol field');
+
+sinon.assert.expose(assert, { prefix: '' });
+
+describe('sealField', () => {
+    const password = 'aPasswordThatHasToBeAtLeast32CharactersLong';
+    const sealed = 'sealed value';
+    let ironMock;
+    let sealField;
+    let object;
```

Comment:
There's a strong preference to not name a variable something generic like `object`. 

1. Best practices -- `object` is subtly one case from being `Object`, which is both a reserved word and is its own object. It's the same reason why we avoid `error` for Errors.


2. It's vague -- It's not very descriptive of what the variable represents. You have to look at how it's being used to figure out what it truly represents, which requires at least one level of indirection when debugging.